doc/nrf/ug_tfm.rst

```diff
@@ -54,12 +58,13 @@ Minimal build
 =============
 
 The default configuration of TF-M has all supported features enabled, which results in a significant memory footprint.
-A minimal version of the TF-M secure application is provided to show how to configure a reduced version of TF-M.
-The non-secure callable interfaces supported by this version generating random numbers, hashing with SHA256, and `tfm_platform_mem_read`.
+A minimal version of the TF-M secure application is provided in |NCS| to show how to configure a reduced version of TF-M.
+The non-secure callable interfaces supported by this version allow for generating random numbers, hashing with SHA-256, and using ``tfm_platform_mem_read``.
```

Comment:
@hakonfam  is this correct understanding? We were missing a verb in this sentence

Reply:
This is correct

Reply:
I think so, yes. 
@hakonfam ^^

Also. Do we need to add "resetting the system from non-secure application"?

Reply:
@oyvindronningstad  Added:
This corresponds to the feature set provided by :ref:`secure_partition_manager`.